Services/Auth0toAI/Auth0toAI/Service/Auth0Service.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+
+namespace Auth0toAI.Service
+{
+    public class Auth0Service
+    {
+        private IConfiguration _config;
+        public TelemetryClient telemetryClient;
+        public Auth0Service(IConfiguration config)
+        {
+            _config = config;
+
+            var telemetryConfiguration = TelemetryConfiguration.CreateDefault();
+            telemetryConfiguration.ConnectionString = _config["Auth0LogConnectionString"];
+            telemetryClient = new TelemetryClient(telemetryConfiguration);
+        }
+
+        public void TrackEventToApplicationInsight(string name, Dictionary<string, string> properties)
+        {
+            telemetryClient.TrackEvent(name, properties);
+            telemetryClient.Flush();
```

Comment:
I'd avoid flushing on each event. let telemetryClient() handle it.

Consider make this Service Disposable to Flush() on Dispose() so that Flush() happens during teardown/stop.